src/components/Toggle/Toggle.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from "react";
+import "./Toggle.css";
+
+function Toggle() {
+  const [isDarkThemeOff, setIsDarkThemeOff] = useState(true);
```

Comment:
do not forget add types when you use useState hook

Reply:
done